rust-code-analysis-cli/src/formats.rs

```diff
@@ -66,7 +66,7 @@ impl Format {
                 .iter()
                 .filter(|v| {
                     if let Some(s) = v.to_str() {
-                        ![".", ".."].contains(&s)
+                        ![".", "..", "/", "\\", ":"].contains(&s)
```

Comment:
These are path components, how can there be `/` or `\\` in them?

Reply:
The [components](https://doc.rust-lang.org/std/path/enum.Component.html#examples) function maintains the root of a path, so if we pass an absolute path `\` on Unix and `C:\` on Windows are maintained.

Reply:
We could remove the root first and then get the components of the resulting relative path

Reply:
Yep :)

Reply:
It is not necessary to remove Windows symbols to create a filename, so we can only remove the root from Linux paths.